app/services/imo_api_service.rb

```diff
@@ -13,13 +14,7 @@ def initialize(message, path: nil, response: nil, exception_message: nil)
     end
   end
 
-  attr_reader :patient
-
-  def initialize(patient)
-    @patient = patient
-  end
-
-  def invite
+  def invite(patient)
```

Comment:
How about `send_invitation` to be consistent with the other method here?

Reply:
Will do.